include/libtorrent/bandwidth_limit.hpp

```diff
@@ -33,30 +33,29 @@ POSSIBILITY OF SUCH DAMAGE.
 #ifndef TORRENT_BANDWIDTH_CHANNEL_HPP_INCLUDED
 #define TORRENT_BANDWIDTH_CHANNEL_HPP_INCLUDED
 
-#include "libtorrent/aux_/disable_warnings_push.hpp"
+#include "libtorrent/assert.hpp"
```

Comment:
I'm just a little bit hesitant because of the includes being re-ordered here. It's probably not a big deal, but it will add a conflict when merging forward most likely